client/vat/form/tasks/vat-number-task.tsx

```diff
@@ -33,6 +39,54 @@ const getVatPrefix = () => {
 	}
 };
 
+const getVatTaxIDName = () => {
```

Comment:
The return value of `getVatTaxIDName()` is the return value of `__()` but I see it will be fed into another `__()`. I was concerned that the string will be double-translated, but I think it will not because it's considered as a parameter for the 2nd `__()` call.

Reply:
This is part of the reason why I would prefer to have separate dialogs for each region (as needed). Then the translator can translate as whole, and [avoid concatenating translated strings (which is a localisation no-no)](https://sapegin.me/blog/internationalization).

Reply:
I think the way it's done here is okay. We're not doing `{phrase1} {tax number} {phrase2}` we're instead doing `Set your %s` (where %s is `tax_number`) which is exactly how that link recommends doing it.
While whole strings would be better, this approach is fine.

Reply:
Yea it's shippable as is, but the amount of branching and substitutions is the problem, _() on _() is a symptom of that.

> While whole strings would be better, this approach is fine.

Exactly, whole strings (or UIs) would be better, my plan is next time we work on this we do that refactor.